routes/product/helpers.js

```diff
@@ -89,15 +108,20 @@ export default {
                         { producerId: { $eq: producerId } },
                     ],
                 };
```

Comment:
what is this filter doing and what is it filtering

Reply:
we are filter products where ` _id is equal to _id and producerId is not null and producerId is equal to producerId`. It is finding exactly the product that we need to updated